src/cli/help.js

```diff
@@ -27,15 +29,18 @@ const help: Command = async (args, std) => {
 };
 
 function usage(print: (string) => void): void {
+  // TODO: Make the usage function pull its list of commands
```

Comment:
Noted. I considered doing something like this when originally designing
the CLI system. I opted not to because it wasn’t clear to me what the
right structure would be. It seems natural to expand the `Command`
interface to require a `help` command… but the help message for each
command is itself a `Command`, and _it_ doesn’t have a help message. The
other consideration was that I wanted to leave the door open to
implement subcommands (for instance, `sourcecred load repo foo/bar` vs.
`sourcecred load org foo`, or `sourcecred cache clear foo/bar` vs.
`sourcecred cache clear-all`).

My assumption was that people creating CLI commands would probably start
by forking or at least referring to one of the existing commands and its
surrounding context, and figure out that a help message should be added.
It’s not foolproof, but it’s worked 100% of the time! (*n* = 1) :-)

Reply:
I actually forgot to add it to the usage string, and in the first code review I believe it wasn't there, and you didn't catch it either. But then I caught it after the first code review. So one could argue that it works 33% of the time. :)